geo/src/main/res/layout/selection_map_layout.xml

```diff
@@ -35,16 +36,16 @@
             android:layout_above="@id/geometry_status"
             android:layout_below="@id/title" />
 
-        <org.odk.collect.androidshared.ui.multiclicksafe.MultiClickSafeImageButton
```

Comment:
Here we used `MultiClickSafeImageButton` probably becasue it was possible to open form filling twice? I think we should protect users from doing that.